### ISSUE TITLE
When I trained the model, I got nan for train and validation loss!

### ISSUE THREAD

**Issue by lihua213** [georgesung/ssd_tensorflow_traffic_sign_detection#33]
When I trained the model, I got nan for train and validation loss! with tensorflow 1.4, python3.5

**Comment by sharpen4** [comment]
When I tried to train more kinds of traffic signs I got nan for train and validation loss. Can you tell me why?

**Comment by youthM** [comment]
I met the problem too,do you solve it? @sharpen4
